ledger/participant-state/kvutils/tools/src/main/scala/ledger/participant/state/kvutils/tools/integritycheck/IntegrityChecker.scala

```diff
@@ -327,7 +321,7 @@ object IntegrityChecker {
           case Failure(exception) =>
             exception.printStackTrace()
             sys.exit(1)
-        }(DirectExecutionContext)
+        }
```

Comment:
Should you have used `parasitic` here?

Reply:
I don't see why it ever used `DirectExecutionContext` when there's a perfectly good thread pool available.